README.md

```diff
@@ -109,45 +127,118 @@ Amoro contains modules as below:
+|-----------------------|-------------------------------------------------|-----------------------------------------------------------|  
+| Standard JDK 8 Build  | `mvn clean package -Pjava8`                     | Build all modules (except `amoro-mixed-trino`) with JDK 8 |  
+| JDK 8 with Hadoop 2.x | `mvn clean package -DskipTests -Pjava8,hadoop2` | Build with JDK 8 and Hadoop 2.x dependencies              |  
+| JDK 8 Quick Build     | `mvn clean package -DskipTests -Pjava8`         | Skip tests and build with JDK 8                           |  
+
+#### Building with JDK 11
+
+JDK 11 is the default compilation version, so no special profile is needed:
+
+| Build Scenario         | Command                                   | Description                                                |  
+|------------------------|-------------------------------------------|------------------------------------------------------------|  
+| Standard JDK 11 Build  | `mvn clean package`                       | Build all modules (except `amoro-mixed-trino`) with JDK 11 |  
+| JDK 11 with Hadoop 2.x | `mvn clean package -DskipTests -Phadoop2` | Build with JDK 11 and Hadoop 2.x dependencies              |  
+| JDK 11 Quick Build     | `mvn clean package -DskipTests`           | Skip tests and build with JDK 11                           |  
+
+#### Building with JDK 17
+
+For JDK 17, you can build all modules including `amoro-mixed-trino`:
+
+| Build Scenario         | Command                                                                                                                             | Description                                   |  
+|------------------------|-------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------|  
+| JDK 17 All Modules     | `mvn clean package -DskipTests -Ptoolchain,build-mixed-format-trino`                                                                | Build all modules including Trino with JDK 17 |  
+| JDK 17 Trino Only      | `mvn clean package -DskipTests -Pformat-mixed-format-trino,build-mixed-format-trino -pl 'amoro-format-mixed/amoro-mixed-trino' -am` | Build only Trino module with JDK 17           |  
+| JDK 17 with Hadoop 2.x | `mvn clean package -DskipTests -Ptoolchain,build-mixed-format-trino,hadoop2`                                                        | Build all modules with JDK 17 and Hadoop 2.x  |  
+
+### Basic Build Options
+
+The most commonly used build commands:
+
+| Build Scenario      | Command                                    | Description                                    |  
+|---------------------|--------------------------------------------|------------------------------------------------|  
+| Standard Build      | `mvn clean package`                        | Build all modules (except `amoro-mixed-trino`) |  
```

Comment:
Need to explain the default configuration, such as HADOOP version, SPARK, FLINK version, etc.